tests/hardware_config_qx.json

```diff
@@ -227,14 +227,13 @@
         "disable_optimization": true
         },
 
-   "display_binary": {
-      "duration": 20,
-      "qubits": [],
-      "matrix": [ [0.0,1.0], [1.0,0.0],
-                  [1.0,0.0], [0.0,0.0] ],
-      "disable_optimization": true
+      "display_binary": {
```

Comment:
I'm assuming this file should not be touched in this PR?

Reply:
No, indeed.